AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -72,24 +72,35 @@ private extension SessionHeaderView {
             //  |_________|     |-------------------|
             // so the idea at leat for now is this below
             #warning("Fix - Handle session.deviceType (for now it is always nill)")
-            switch session.type?.description {
+            sensorType
+                .font(Font.moderate(size: 13, weight: .regular))
+        }
+        .foregroundColor(.darkBlue)
+    }
+    
+    var sensorType: some View {
+        switch session.type.description {
             case "Fixed":
-                Text("\(session.type!.description) : AirBeam3")
-                    .font(Font.moderate(size: 13, weight: .regular))
+                return airBeamText
             case "Mobile":
-                if session.allStreams!.count > 1 {
-                    Text("\(session.type!.description): AirBeam3")
-                        .font(Font.moderate(size: 13, weight: .regular))
+                if session.allStreams!.count == 1 {
+                    return Text("\(session.type!.description): \(Strings.SessionHeaderView.mic)")
                 } else {
-                    Text("\(session.type!.description): Phone Mic")
-                        .font(Font.moderate(size: 13, weight: .regular))
+                    return airBeamText
                 }
             default:
-                Text(session.deviceType?.description ?? "")
-                    .font(Font.moderate(size: 13, weight: .regular))
-            }
+                return Text(session.deviceType?.description ?? "")
+        }
+    }
+    
+    var airBeamText: Text {
+        if session.pm1Stream?.sensorName?.contains(Strings.SessionHeaderView.airBeam1) != nil {
```

Comment:
that's kind of weird expression ;D 
What would you say about adding device types here and then using them in this expression?  

```
public enum DeviceType: Int, CustomStringConvertible {
    case MIC = 0
    case AIRBEAM3 = 1

    public var description: String {
        switch self {
        case .MIC: return "Device's Microphone"
        case .AIRBEAM3: return "AirBeam 3"
        }
    }
}
```

Reply:
correct me if I'm wrong but I think you'd had to check the integers for AB1 and AB2 in android code